R/borel.r

```diff
@@ -27,19 +27,14 @@ dborel <- function(x, mu, log = FALSE) {
 ##' Random numbers are generated by simulating from a Poisson branching process
 ##' @param n Number of random variates to generate.
 ##' @inheritParams dborel
-##' @param infinite A threshold above which the random number is set to
-##'  `Inf` as a form of truncation. This is important because `rborel()` uses
-##'  `simulate_summary()` internally with `offspring_dist = rpois`, so
-##'  when `mu >= 1`, the simulation could grow out of proportion and chain
-##'  summaries (the random numbers generated here) need to be truncated using
-##'  `infinite`.
+##' @inheritParams simulate_summary
```

Comment:
If doing this we'll have to change the documentation of the argument in `simulate_summary()` as it refers to a `statistic` argument which does not exist here.

Perhaps just rename it to `max` and keep the documentation as is?

Reply:
That's true. I've renamed it to `censor_at` in 55e8cfed89fc8b7e3b35ea77f0e8d9c257da4a78 and clarified the documentation to reflect the fact that values are replaced with `Inf` at the given censoring limit and simulations are stopped.